backend/danswer/db/index_attempt.py

```diff
@@ -184,37 +182,19 @@ def get_last_attempt(
 
 
 def get_latest_index_attempts(
-    connector_credential_pair_identifiers: list[ConnectorCredentialPairIdentifier],
     secondary_index: bool,
     db_session: Session,
 ) -> Sequence[IndexAttempt]:
     ids_stmt = select(
         IndexAttempt.connector_credential_pair_id,
-        func.max(IndexAttempt.time_created).label("max_time_created"),
+        func.max(IndexAttempt.id).label("max_id"),
     ).join(EmbeddingModel, IndexAttempt.embedding_model_id == EmbeddingModel.id)
 
     if secondary_index:
         ids_stmt = ids_stmt.where(EmbeddingModel.status == IndexModelStatus.FUTURE)
     else:
         ids_stmt = ids_stmt.where(EmbeddingModel.status == IndexModelStatus.PRESENT)
 
-    where_stmts: list[ColumnElement] = []
-    for connector_credential_pair_identifier in connector_credential_pair_identifiers:
```

Comment:
this was not necessary - we always fetch all of them anyways. Just added complexity / slowness